ql/ql/src/queries/style/OverrideAny.ql

```diff
@@ -0,0 +1,40 @@
+/**
+ * @name Override with unmentioned parameter
+ * @description A predicate that overrides the default behavior but doesn't mention a parameter is suspicious.
+ * @kind problem
+ * @problem.severity warning
+ * @id ql/override-any
+ * @precision very-high
```

Comment:
Since the warnings are flagging benign cases, is precision `very-high` appropriate?

Reply:
Maybe not, but it doesn't really matter.  
If we get FP reports (I doubt it), then we can fix it.

Reply:
I lowered it to `high` since I touched the query anyway.